qiskit_nature/properties/electronic/electronic_energy.py

```diff
@@ -0,0 +1,120 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The ElectronicEnergy property."""
+
+from typing import cast, Dict, List, Optional, Union
+
+from qiskit_nature import QiskitNatureError
+from qiskit_nature.drivers import QMolecule, WatsonHamiltonian
+from qiskit_nature.operators.second_quantization import FermionicOp
+from qiskit_nature.results import EigenstateResult
+
+from .bases import ElectronicBasis
+from .integrals import (
+    ElectronicIntegrals,
+    OneBodyElectronicIntegrals,
+    TwoBodyElectronicIntegrals,
+)
+from ..property import Property
+
+
+class ElectronicEnergy(Property):
```

Comment:
I proposed before the change of electronic to electronic_structure, but for this property I would keep it as ElectronicEnergy.

Reply:
Leaving this open for information purposes. Nothing to be resolved here, unless someone disagrees.